### ISSUE TITLE
question about rotation sliders

### ISSUE THREAD

**Issue by JoOkuma** [GuignardLab/napari-manual-registration#2]
hi, 

First of all, I would like to thank you for making this great plugin available.

I was wondering if there's any reasoning/benefit for applying the rotation to the data directly instead of at the `rotate` parameters when using the sliders.

https://github.com/GuignardLab/napari-manual-registration/blob/af51a376424756fb57b4c4519dd30e7d2050efbe/src/napari_manual_registration/_widget.py#L386-L397

I'm using this plugin while aligning multiple layers, and this approach disallows using `link_layers` from napari.

Thanks,
Jordao

**Comment by jules-vanaret** [comment]
Hi @JoOkuma, thank you very much for your feedback. 
The reason we don't use the `layer.rotate` parameter is because we designed the plugin to allow the user to switch at all times between the 2D and 3D views. When I wrote the plugin some while back, applying a rotation through `.rotate` then switching to 2D gave the warning message:
```
Non-orthogonal slicing is being requested, but is not fully supported. Data is displayed without applying an out-of-slice rotation or shear component.
``` 
which led to data being distorted and partly excluded from the slider range.

I try to keep up with new Napari updates but if I'm not mistaken, this feature has not been implemented yet, hence the brute force solution of rotating the array.

If you are aware of any solution involving `layer.rotate` I am extremely interested !
